include/librealsense2/h/rs_types.h

```diff
@@ -221,6 +221,7 @@ typedef enum rs2_extension
     RS2_EXTENSION_MAX_USABLE_RANGE_SENSOR,
     RS2_EXTENSION_DEBUG_STREAM_SENSOR,
     RS2_EXTENSION_CALIBRATION_CHANGE_DEVICE,
+    RS2_EXTENSION_SAFETY_PRESET,
```

Comment:
I don't think a safety preset is an extension.
Extension means it inherent a device/sensor and add capabilities to it.
Speak with Remi, maybe he is about to add a safety device/sensor capability and the new API's can be exposed through it

Reply:
Nir is right - there is such extension for the sensor

Reply:
Take a look at `max_usable_range_sensor`
We can say that the safety sensor extend the sensor and expose new functions\
https://github.com/IntelRealSense/librealsense/blob/923253f645e867c75e3c6c30a84127f41d82f3e9/include/librealsense2/hpp/rs_sensor.hpp#L728

Reply:
and an extension should be placed here too IMO:
https://github.com/IntelRealSense/librealsense/blob/923253f645e867c75e3c6c30a84127f41d82f3e9/src/rs.cpp#L1482